test/Features/Bootstrap/FeatureContext.php

```diff
@@ -56,8 +56,36 @@ public function gatherContexts(BeforeScenarioScope $scope) {
    */
   public function iExpectYouToDie()
   {
+
+print <<<END
+
+       _.-._
+      ({  `}) *WHAM!*
+        \_/
+ .(     -|-     ),
+`  -----' `-----  '
+
+END;
     die();
   }
+
+  /**
+   * @Then I want to debug this :node
+   */
+  public function iWantToDebugThisNode($node)
```

Comment:
- [ ] One thing I found while I was working on Trust yesterday (which also uses Behat) is that this functionality already exists in the main DrupalContext:

https://github.com/jhedstrom/drupalextension/blob/master/src/Drupal/DrupalExtension/Context/DrupalContext.php#L426